Socket.php

```diff
@@ -49,21 +49,201 @@
 class Socket extends HoaSocket
 {
     /**
+     * Entity type: isuser.
+     *
+     * @const string
+     */
+    const ENTTYPE_ISUSER = 'isuser';
+
+    /**
+     * Entity type: ischannel.
+     *
+     * @const string
+     */
+    const ENTTYPE_ISCHANNEL = 'ischannel';
+
+    /**
+     * Host type: isserver.
+     *
+     * @const string
+     */
+    const HOSTTYPE_ISSERVER = 'isserver';
+
+    /**
+     * Host type: isnetwork.
+     *
+     * @const string
+     */
+    const HOSTTYPE_ISNETWORK = 'isnetwork';
+
+    /**
+     * Entity.
+     * @var string
+     */
+    private $_entity;
+
+    /**
+     * Entity type.
+     * @var string
+     */
+    private $_entityType = self::ENTTYPE_ISCHANNEL;
+
+    /**
+     * Host type.
+     * @var string
+     */
+    private $_hostType;
```

Comment:
Any reason to put everything in the private scope? By default, we prefer to use the protected scope, except strong reasons.